### DISCUSSION TITLE
Uploading files for releases is extremely slow, and fails (likely due to timing out) before it can complete

### DISCUSSION THREAD

**Discussion by tabemann** [community/community#144590] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am trying to upload a 25 MiB `.tar.gz` file for a new release, and it uploads extremely slowly compared to the performance I normally get with files of such size, such that it fails long before it would complete, likely due to timing out. Note that I do not have any other Internet problems at the present ─ uploading the same exact file to hackaday.io and to Facebook (because I could not upload it here) went without any issues.

**Comment by kivanc57** [comment]
_It's not a bug_ but git has its own limitations for files being uploaded. I know it because I had to go over a similar scenerio. All you can to  to use a third service, such as cloud and then sharing the link in your repository or just using a mechanical drive such as USB which is the securest way for me.

**Comment by tabemann** [comment]
On Wed, Nov 27, 2024 at 4:48 AM Kivanc Gordu ***@***.***>
wrote:

> *It's not a bug* but git has its own limitations for files being
> uploaded. I know it because I had to go over a similar scenerio. All you
> can to to use a third service, such as cloud and then sharing the link in
> your repository or just using a mechanical drive such as USB which is the
> securest way for me.
>
> This was not uploading files with git but rather with uploading files with
GitHub's releases service from the web browser.

Travis
